File: Data Structures & Algorithms/number-of-sub-arrays-of-size-k-and-average-greater-than-or-equal-to-threshold/submission-0.java
class Solution {
    public int numOfSubarrays(int[] arr, int k, int threshold) {
        int l=0,n=arr.length;
        int ans=0;
        while(l<n-k+1){
            int sum=arr[l];
            for(int r=l+1;r-l<k && r<n;r++){
                sum=sum+arr[r];


            }
            if(sum/k>=threshold){ans++;}
            l++;
        }
       return ans; 
    }
}

File: Data Structures & Algorithms/non-cyclical-number/submission-0.py
class Solution:
    def isnoncylinder(self,n:int )-> int :
        sum=0
        while n>0:
            sum=sum+(n%10)**2
            n=n//10
        return sum
    def isHappy(self, n: int) -> bool:
        ans=[]
        while n!=1 and n not in ans:
            ans.append(n)
            n=self.isnoncylinder(n)
        return n==1
        


        